### PR TITLE
chore: Automate releases.

### PR DESCRIPTION
**Pull request by lukesneeringer**
This adds release automation from GitHub, so I can make a `vX.Y.Z` tag and the release notes populate automatically and a release goes to PyPI.

We previously were not using PyPI for releases (preferring downloading a ZIP from GitHub instead), but moving to PyPI means I can use Renovate on the AIP repos, which is valuable because it allows me to "self-approve" version bumps that come from the bot.

### CODE REVIEW

**Comment by lukesneeringer** [comment]
As always with these kinds of PRs, there is about to be some thrashing. :-)

**Comment by lukesneeringer** [comment]
@jgeewax This now is ready and actually works.

File: .github/release_notes.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    proc = subprocess.run(cmd, capture_output=True, text=True)
+    if proc.returncode != 0:
+        print(f'Error running {cmd[0]}: {proc.stderr}', file=sys.stderr)
+        sys.exit(proc.returncode)
+    return proc.stdout.strip()
+
+
+if __name__ == '__main__':
+    # Get the previous tag and the current tag.
+    revs = exec(['git', 'rev-list', '--simplify-by-decoration',
```
Comment (jgeewax) [inline .github/release_notes.py:73]: Let's create an issue to make possibly use a Git Python library rather than the CLI + exec...

**Review by jgeewax — APPROVED** [review]

File: .github/release_notes.py (hunk 1 above)
Comment (lukesneeringer) [reply .github/release_notes.py:73]: #17

**Pull request merged**